src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageClient.java

```diff
@@ -54,7 +54,7 @@ public interface SonarLintExtendedLanguageClient extends LanguageClient {
   void showRuleDescription(ShowRuleDescriptionParams params);
 
   @JsonNotification("sonarlint/showHotspot")
-  void showHotspot(ServerHotspot hotspot);
+  void showHotspot(ServerHotspotDetails hotspot);
```

Comment:
I realize that we are relying on the type from the core and forward it to the client, this might be a bit fragile. Not very urgent to fix though